src/toolchain/export-android.md

```diff
@@ -0,0 +1,105 @@
+<!--
+ ~ Copyright (c) godot-rust; Bromeon and contributors.
+ ~ This Source Code Form is subject to the terms of the Mozilla Public
+ ~ License, v. 2.0. If a copy of the MPL was not distributed with this
+ ~ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+-->
+
+# Export to Android
+
+NOTE: Keep track of where you install the Android Command Line Tools and JDK 17
+
+Exporting with gdext for Godot requires some of the same pieces that are required for building Godot from source.
+Specifically, the Android SDK Command Line Tools and JDK 17 as mentioned in Godot's documentation
+[here](https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#requirements)
+
+Once you have those installed, you then need to follow Godot's instructions for setting up the build system
+[here](https://docs.godotengine.org/en/stable/contributing/development/compiling/compiling_for_android.html#setting-up-the-buildsystem)
+
+As of writing this, it is not entirely clear if JDK 17 is required in order for Android Command Line Tools to function properly.
+It has not been not tested without it. To be safe, it is included in the documentation for now.
+
+
+## Compiling
+
+The environment variable `CLANG_PATH` is used by bindgen's clang-sys dependency. See also
+[clang-sys documentation](https://github.com/KyleMayes/clang-sys?tab=readme-ov-file#environment-variables)
+
+Set the environment variable `CLANG_PATH` to point to Android's build of clang. Example:
+
+```bash
+export CLANG_PATH = "{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/{hostMachineOs}/bin/clang"
+```
+
+Then set the `CARGO_TARGET_{targetTripleHere}_LINKER` to point to the Android linker for the Android triple you are targeting.
+You need to compile your gdext library for each Android triple individually. Possible targets can be found by running:
+
+```bash
+rustup target list
+```
+
+You can find the linkers in the Android CLI directory at:
+
+```text
+"{androidCliDirectory}/{androidCliVersion}/ndk/{ndkVersion}/toolchains/llvm/prebuilt/{hostMachineOs}/bin/{targetTripleHere}{androidVersion}"
+```
```

Comment:
Also this could be split (without `\`, at one of the `/` points). 

And the quotes are not needed.

`targetTripleHere` could be just `targetTriple`.